rainforest-cli.go

```diff
@@ -246,6 +246,10 @@ func main() {
 					Name:  "wait, reattach",
 					Usage: "monitor existing run with `RUN_ID` instead of starting a new one.",
 				},
+				cli.BoolFlag{
```

Comment:
What about using an `IntFlag` (`max-retries` ?) instead? That we'd default to `0`.